test/parallel/test-console-log-stdio-broken-dest.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const { Writable } = require('stream');
+const { Console } = require('console');
+
+const stream = new Writable({
+  write(chunk, enc, cb) {
+    cb();
+  },
+  writev(chunks, cb) {
+    setTimeout(cb, 10, new Error('kaboom'));
+  }
+});
+const myConsole = new Console(stream, stream);
+
+process.on('warning', common.mustNotCall);
+
+stream.cork();
+for (let i = 0; i < 20; i++) {
```

Comment:
Maybe the loop limit can be something like `EventEmitter.defaultMaxListeners + 1` instead of a hardcoded value?

Reply:
updated